cmd/traefik/traefik.go

```diff
@@ -206,6 +207,13 @@ func run(traefikConfiguration *server.TraefikConfiguration) {
 	}
 	log.SetLevel(level)
 	if len(globalConfiguration.TraefikLogsFile) > 0 {
+		dir := filepath.Dir(globalConfiguration.TraefikLogsFile)
+
+		err := os.MkdirAll(dir, 0755)
+		if err != nil {
+			log.Errorf("Failed to create log path %s: %s", dir, err)
+		}
+
```

Comment:
The file seems completely untested, but making it properly testable is likely a bigger refactoring we want to source out to a different PR.

Could you verify manually that this part of the code change works as expected please?